app/features/TypeData.py

```diff
@@ -0,0 +1,40 @@
+import pandas as pd
+import numpy as np
+
+
+class TypeData:
+    def __init__(self, nameType):
+        self.dataFrame = None
+        self.data = []
+        self.nameType = nameType
+
+    def getRepetition(self):
+        df = pd.DataFrame({'percentage': self.dataFrame[self.nameType].value_counts(normalize=True) * 100, 'count': self.dataFrame.pivot_table(index=[self.nameType], aggfunc='size')})
+        return df
+
+    def get(self):
+        return self.dataFrame
+
+    def append(self, note, currentTime):
+        self.data.append([note, currentTime])
+
+    def getAverageByPartsTime(self, nbSplit):
+        arrayAverage = []
+        timeToSplit = self.dataFrame['currentTime'].iloc[-1] / nbSplit
+        i = 0
+
+        while i < nbSplit:
+            arrayAverage.append(round(pd.DataFrame(self.dataFrame[(self.dataFrame['currentTime'] <= timeToSplit * (i + 1)) & (self.dataFrame['currentTime'] >= timeToSplit * i)])[self.nameType].mean(), 2))
```

Comment:
```suggestion
            arrayAverage.append(round(self.dataFrame[(self.dataFrame['currentTime'] < timeToSplit * (i + 1)) & (self.dataFrame['currentTime'] >= timeToSplit * i)][self.nameType].mean(), 2))
```

Reply:
I need to leave pd.DataFrame to get the indexes I want